app/src/models/hash.ts

```diff
@@ -0,0 +1,5 @@
+type HashableType = number | string | boolean | undefined | null
```

Comment:
```suggestion
/**
  Types which can safely be coerced to strings without loosing information
   As an example `1234.toString()` doesn't loose any information whereas
   `({ foo: bar }).toString()` does (`[Object object]`)
**/
type HashableType = number | string | boolean | undefined | null
```

Reply:
Is the use of "loosing" and "loose" intentional or you meant "losing" and "lose"? 🤔